metron-stellar/stellar-common/src/main/java/org/apache/metron/stellar/common/shell/StellarResult.java

```diff
@@ -0,0 +1,185 @@
+/*
+ *
+ *  Licensed to the Apache Software Foundation (ASF) under one
+ *  or more contributor license agreements.  See the NOTICE file
+ *  distributed with this work for additional information
+ *  regarding copyright ownership.  The ASF licenses this file
+ *  to you under the Apache License, Version 2.0 (the
+ *  "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package org.apache.metron.stellar.common.shell;
+
+import java.util.Optional;
+
+/**
+ * The result of executing a Stellar expression within a StellarShellExecutor.
+ */
+public class StellarResult {
```

Comment:
Instead of the Stellar executor just returning an Object like before (which doesn't tell me much about whether the operations was successful or not) I needed a more descriptive result like this.  

This is how the CLI and Zeppelin REPLs determine whether the Stellar expression was executed successfully or not.  They need to perform different actions based on success, failure or even if there is a termination request.

Reply:
If this is Shell only can we name it StellarShellResult?

Reply:
That's what I had it originally.  Ha.  

It is already in the "shell" package, so I thought making the name simpler is just easier.